MatrixSDK/Data/RoomSummaryStore/CoreData/Models/MXRoomLastMessageModel.swift

```diff
@@ -0,0 +1,61 @@
+// 
+// Copyright 2021 The Matrix.org Foundation C.I.C
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import Foundation
+import CoreData
+
+@objc(MXRoomLastMessageModel)
+public class MXRoomLastMessageModel: NSManagedObject {
+
+    internal static func typedFetchRequest() -> NSFetchRequest<MXRoomLastMessageModel> {
+        return NSFetchRequest<MXRoomLastMessageModel>(entityName: entityName)
+    }
+
+    @NSManaged public var s_eventId: String
```

Comment:
Just a thought, but do all these Core Data properties need to be public? As in, should any of these `MX...Model` classes actually be used from outside of the SDK?

Reply:
Nope, all converted to internal.